code/src/java/pcgen/core/Equipment.java

```diff
@@ -5510,6 +5470,7 @@ public int getCritMultiplier()
 	 * Gets the altCritMultiplier attribute of the Equipment object
 	 * 
 	 * @return The altCritMultiplier value
+	 * @deprecated due to CRITMULT code control
 	 */
 	public int getAltCritMultiplier()
```

Comment:
If you are going to have @deprecation in the method javadoc, wouldn't you also want to add 
@Deprecated to the method as well?